src/components/structures/RightPanel.tsx

```diff
@@ -296,7 +295,13 @@ export default class RightPanel extends React.Component {
                 break;
 
             case RightPanelPhases.NotificationPanel:
-                panel = <NotificationPanel onClose={this.onClose} />;
+                if (SettingsStore.getValue("feature_pinning")) {
+                    panel = <NotificationPanel onClose={this.onClose} />;
+                }
+                break;
+
+            case RightPanelPhases.PinnedMessages:
+                panel = <PinnedMessagesCard room={this.props.room} onClose={this.onClose} />;
```

Comment:
I'm failing to understand what this change is really doing: if the labs flag is *disabled*, we refuse to show the notifications panel now? These two panels seem unrelated in nature.

Reply:
Yeah I put the if around the wrong thing :D